client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -509,8 +509,8 @@ function recordOrderInGAEnhancedEcommerce( cart, orderId, wpcomJetpackCartInfo )
 function recordOrderInJetpackGA( cart, orderId, wpcomJetpackCartInfo ) {
 	if ( wpcomJetpackCartInfo.containsJetpackProducts ) {
 		fireEcommercePurchaseGA4(
-			cartToGaPurchase( orderId, cart, wpcomJetpackCartInfo, TrackingEnvironment.JETPACK ),
-			TrackingEnvironment.JETPACK
+			cartToGaPurchase( orderId, cart, wpcomJetpackCartInfo ),
+			Ga4PropertyGtag.WPCOM
```

Comment:
```suggestion
			Ga4PropertyGtag.JETPACK
```

I think it might be a mistake

Reply:
Yes, definitely. Updating. :)